script.js

```diff
@@ -0,0 +1,31 @@
+function sum1() {
+  var num1 = document.getElementById('num1').value
+  var num2 = document.getElementById('num2').value
+  var outputElement = document.getElementById('output1')
+
+  outputElement.innerHTML = parseInt(num1) + parseInt(num2)
+}
+
+function sum2() {
+  var num3 = document.getElementById('num3').value
+  var num4 = document.getElementById('num4').value
+  var outputElement = document.getElementById('output2')
+
+  outputElement.innerHTML = parseInt(num3) - parseInt(num4)
+}
+
+function sum3() {
+  var num5 = document.getElementById('num5').value
+  var num6 = document.getElementById('num6').value
+  var outputElement = document.getElementById('output3')
+
+  outputElement.innerHTML = parseInt(num5) * parseInt(num6)
+}
+
+function sum4() {
+  var num7 = document.getElementById('num7').value
+  var num8 = document.getElementById('num8').value
+  var outputElement = document.getElementById('output4')
+
+  outputElement.innerHTML = parseInt(num7) / parseInt(num8)
+}
```

Comment:
This code becomes repetitive because of the choice to use a different function for each operation. It could be done with a single form and a single function.

```HTML
<input type="text" id="num1" />
<select id="operator">
  <option value="+">+</option>
  <option value="-">-</option>
  <option value="*">*</option>
  <option value="/">/</option>
</select>
<input type="text" id="num2" />
<button onclick="calculate()">=</button>
```

```JavaScript
function calculate() {
  var num1 = document.getElementById('num1').value
  var num2 = document.getElementById('num2').value
  var operator = document.getElementById('operator').value
  var outputElement = document.getElementById('output4')
  var result

  switch(operator) {
    case '+':
      result = num1 + num2
      break
    case '-':
      result = num1 + num2
      break
    case '*':
      result = num1 + num2
      break
    case '/':
      result = num1 + num2
      break
  }

  outputElement.innerHTML = result
}
```